### PR TITLE
WPPB New extension

### PR DESCRIPTION
**Pull request by pernielsentikaer**
## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
@jfkisafk feel free to have a look 🙂

**Comment by jfkisafk** [comment]
For sure @pernielsentikaer , I will take a look tomorrow

File: extensions/wppb/src/create-boilerplate.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        placeholder="Plugin Description"
+        info="This is my awesome extension"
+        {...itemProps.pluginDescription}
+      />
+      <Form.FilePicker
+        title="Output Directory"
+        {...itemProps.outputDirectory}
+        canChooseFiles={false}
+        allowMultipleSelection={false}
+        canChooseDirectories={true}
```
Comment (jfkisafk) [inline extensions/wppb/src/create-boilerplate.tsx:128]:
```suggestion
        canChooseDirectories
```

File: extensions/wppb/src/create-boilerplate.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      <Form.TextField title="Plugin Name" placeholder="My Awesome Extension" {...itemProps.pluginName} />
+      <Form.TextField title="Plugin Slug" placeholder="awesome-extension" {...itemProps.pluginSlug} />
+      <Form.TextField title="Plugin URL" placeholder="https://example.com" {...itemProps.pluginURL} />
+      <Form.TextField title="Author Name" storeValue placeholder="Jill Doe" {...itemProps.authorName} />
+      <Form.TextField title="Author Email" storeValue placeholder="jill@example.doe" {...itemProps.authorEmail} />
+      <Form.TextField title="Author URL" storeValue placeholder="https://jill.doe" {...itemProps.authorURL} />
+      <Form.TextArea
+        title="Plugin Description"
+        placeholder="Plugin Description"
+        info="This is my awesome extension"
```
Comment (jfkisafk) [inline extensions/wppb/src/create-boilerplate.tsx:120]: Shouldn't this be the other way around?
```suggestion
        info="Plugin Description"
        placeholder="This is my awesome extension"
```

**Review by jfkisafk — APPROVED** [review]
LGTM, just a couple of minor comments

**Pull request merged**